Tests/FunctionalTests/helpermethod.ts

```diff
@@ -147,3 +156,62 @@ export async function AcceptCookies(page: Page) {
     }
   }
 }
+
+export async function ExpectAllResultsHaveBatchUserAttValue(
```

Comment:
There's a lot of Assertion helper methods, I would like to see these put into their own file

Reply:
One for another time. Sorry